sender-kinesis/src/main/java/zipkin/reporter/kinesis/KinesisSender.java

```diff
@@ -68,6 +69,9 @@ public static Builder builder() {
     /** Maximum size of a message. Kinesis max message size is 1MB */
     Builder messageMaxBytes(int messageMaxBytes);
 
+    /** Allows you to change to json format. Default is thrift */
+    Builder encoding(Encoding encoding);
```

Comment:
note: "encoding" is thrift or json, the "encoder" says how to encode in thrift or json. Soon, we'll have Encoder.JSON2, but this won't affect code in this class (though we might use that encoder in tests)